schemas/context/namespace.schema.json

```diff
@@ -9,26 +9,18 @@
   "$schema": "http://json-schema.org/draft-06/schema#",
   "title": "Namespace",
   "type": "object",
-  "description":
-    "The namespace or unique identifier associated with a collection of data. Each EndUserID is associated with a given namespace. A user can create and obtain information about a namespace during the solution onboarding.\n",
-  "definitions": {
+  "description": "",
+    "definitions": {
     "namespace": {
       "properties": {
-        "@id": {
-          "title": "Identifier",
-          "type": "string",
-          "format": "uri",
-          "description":
-            "The numeric ID associated with this namespace. This would be provided by the individual or system that created the namespace.\n"
-        },
         "xdm:code": {
           "title": "Code",
           "type": "string",
           "description":
-            "The code is a shortcut to the full @id and at least one of the code or @id can be used. Sometimes, this code refered to as the data source integration code."
+            "The code is a human readable identifier for the namespace and can be used to request the technical namespace id which is used for identity graph processing."
```

Comment:
The technical namespace ID is referenced in the description, but not visible in the schema.

How is uniqueness of the `code` ensured?

Who governs the creation of new `code` values?